lib/presentation/pages/categories/controller/category_controller.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:get/get.dart';
+import 'package:task/app/utils/custom_strings.dart';
+import 'package:task/data/models/products/product_model.dart';
+import 'package:task/presentation/pages/home_screen/controller/home_controller.dart';
+
+class CategoryController extends GetxController {
+  @override
+  void onInit() {
+    homeController = Get.put(HomeController());
+    super.onInit();
+  }
+
```

Comment:
Take this values from api  not hard coded string and implement https://fakestoreapi.com/products/categories api for getting the values